pypeit/pypeit.py

```diff
@@ -338,6 +348,7 @@ def reduce_all(self):
             # Loop on unique comb_id
             u_combid = np.unique(self.fitstbl['comb_id'][grp_science])
             for j, comb_id in enumerate(u_combid):
+                #import pdb; pdb.set_trace()
```

Comment:
Can you remove this. Have you tried 

from IPython import embed; embed()

A much more fun way to debug in my opinion;)

Reply:
FIxed. Sorry about that, I have a bad habit of putting these in code and forgetting to remove them. Thanks for finding this.